vNext/src/ClientGenerator/BootstrapCodegen.proj

```diff
@@ -1,39 +1,79 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
```

Comment:
You are using a very old SDK... There is no need for those imports anymore.

In fact, if you mix the projects, VS and MSBuild will load assemblies in a wrong way which will mess everything...